extension/src/popup/components/account/AccountAssets/index.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from "react";
 import { useSelector } from "react-redux";
-
 import { BigNumber } from "bignumber.js";
+import SimpleBar from "simplebar-react";
```

Comment:
❓ sgtm to use, I haven't used before though so jc. Is it used for simpler scroll styling?

Reply:
It does 2 things that are pretty useful for us: 1) it overlays the scrollbar on top of your content without pushing the layout (as a built in div scrollbar does) and 2) it disappears when you're not hovering

These are both good for making the UI match the figma design, regardless of content length. The normal implementation of a div scroll bar just causes so many headaches because it has `display: block` and pushes your content around depending on if a scrollbar is necessary or not

Reply:
ahh I see nifty!